internal/pkg/cli/deploy/lbws.go

```diff
@@ -262,7 +262,15 @@ func (d *lbWebSvcDeployer) validateNLBRuntime() error {
 		logAppVersionOutdatedError(aws.StringValue(d.lbMft.Name))
 		return err
 	}
-	return validateLBWSAlias(d.lbMft.NLBConfig.Aliases, d.app, d.env.Name)
+	if err := validateLBWSAlias(d.lbMft.NLBConfig.PrimaryRoutingRule.Aliases, d.app, d.env.Name); err != nil {
```

Comment:
Maybe `d.lbMft.NLBConfig.Aliases()`?

Reply:
I think we need to write a `Aliases()` on `NLBConfig` struct in order to do what you are suggesting.